mono/mini/exceptions-arm.c

```diff
@@ -14,7 +14,7 @@
 #include <string.h>
 
 #ifndef MONO_CROSS_COMPILE
-#ifdef HAVE_ASM_SIGCONTEXT_H
+#if defined(HAVE_ASM_SIGCONTEXT_H) && defined(__GLIBC__)
 #include <asm/sigcontext.h>
```

Comment:
I have to be really tired today! 
The meaning of the patches is to NOT include <asm/sigcontext.h> because it's not needed on MUSL. Here I'm simply skipping including <asm/sigcontext.h> if we are on musl cause `__GLIBC__` is not defined.